test/e2e/common/host_path.go

```diff
@@ -133,6 +133,7 @@ func mount(source *api.HostPathVolumeSource) []api.Volume {
 func testPodWithHostVol(path string, source *api.HostPathVolumeSource) *api.Pod {
 	podName := "pod-host-path-test"
 
+	priv := true
```

Comment:
I really don't want to run the container privileged if we can avoid it. Is there any other way to do this? Can we create a temporary directory, chcon it to svirt_sandbox_file_t, and then run these tests?